gramps/gen/const.py

```diff
@@ -114,15 +114,16 @@
         USER_HOME = get_env_var("HOME")
     HOME_DIR = get_env_var("SAFEMODE")
 
-
-if os.path.exists(HOME_DIR) or "GRAMPSHOME" in os.environ or "SAFEMODE" in os.environ:
+USER_DATA = os.path.join(GLib.get_user_data_dir(), "gramps")
+USER_CONFIG = os.path.join(GLib.get_user_config_dir(), "gramps")
+USER_CACHE = os.path.join(GLib.get_user_cache_dir(), "gramps")
+if os.path.exists(USER_DATA) and os.path.exists(USER_CONFIG):
```

Comment:
This condition could be merged into elif later on but readability/understandability is lost, so I left it this way with a pass statement.